_sass/components/_footer.scss

```diff
@@ -0,0 +1,154 @@
+.c-footer {
+  background-color: $dark-blue;
+  width: 100%;
+
+  &__container {
+    width: 80%;
+    margin: 0 auto;
+    padding: 4rem 0;
+  }
+
+  &__award-container {
+    width: 100%;
+    padding: 4rem 0;
+  }
+
+  &__award-image {
+    width: 5rem;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__award-text {
+    @include uppercase-white-font;
+    text-align: center;
+    margin: 1.5rem auto 0;
+    max-width: 23.625rem;
```

Comment:
I think the max-width on this could be removed. I left a comment below about another max-width set on a paragraph, and that for a lot of web designs it's okay that the text breaks don't exactly reflect the mockup or original designs on all screen sizes. It will help the design be more flexible across screen sizes 🙂

Reply:
Okay! I'm a control freak. It's in my design nature